tests/integration/sidebar_requirer_tester_charm/metadata.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+name: kubeflow-dashboard-requirer-mock
```

Comment:
Do we want to add doc links here?
```
website:
source:
issues:
```

Reply:
I vote no, just because this isn't a charm that will get published.  It is only used in the integration tests here.  What do you think?